packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021, salesforce.com, inc.
+ * Copyright (c) 2025, salesforce.com, inc.
```

Comment:
```suggestion
 * Copyright (c) 2021, salesforce.com, inc.
```

We keep this as is because this references the year the file was created

Reply:
Alright, made the change